backend/account/migrations/0001_initial.py

```diff
@@ -0,0 +1,50 @@
+# Generated by Django 4.1.5 on 2023-02-08 01:55
+
+import django.contrib.auth.models
+import django.contrib.auth.validators
+from django.db import migrations, models
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        ('auth', '0012_alter_user_first_name_max_length'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='User',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('password', models.CharField(max_length=128, verbose_name='password')),
```

Comment:
Password should be handled entirely by OAuth right? Is this field used anywhere?

Reply:
Also not sure where `is_superuser`, or `is_staff`would be used - got any examples?

Reply:
Those are both from Django's `AbstractUser` class and aren't currently in use in this project. I personally think we should keep the `AbstractUser` inheritance because it has a bunch of convenient helper methods, but if we decide we don't want to have those unused fields then we can just reimplement `AbstractUser`'s helper methods as needed in our own `User` class.

Reply:
Ah sure if if they're inherited doesn't really matter to me if we keep them 👍